source/Octo/Octo.csproj

```diff
@@ -17,11 +17,15 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netcoreapp2.0' ">
```

Comment:
The runtime identifiers were missing for `netcoreapp2.0`

Reply:
What are we even building `netcoreapp2.x`?

Reply:
Ah I see, next file. The runtimes should only be needed for the project that is the executable

Reply:
Looks like we are according to the build file. Sounds like we will phase it out over time: https://github.com/OctopusDeploy/OctopusCLI/blob/master/build.cake#L144

Reply:
Ah, the `.portable` build. We shouldn't need the runtimes then.